src/logic/calculate.test.js

```diff
@@ -0,0 +1,69 @@
+import calculate from './calculate';
+
+describe('calculate', () => {
+  it('returns an empty object when given "AC"', () => {
+    expect(calculate({ total: '5', next: '2', operation: '+' }, 'AC')).toEqual({
+      total: null,
+      next: null,
+      operation: null,
+    });
+  });
+
+  it('updates next when given a number', () => {
+    expect(calculate({ total: '5', next: '2', operation: '+' }, '3')).toEqual({
+      total: '5',
+      next: '23',
+      operation: '+',
+    });
+  });
+
+  it('updates next when given "0"', () => {
+    expect(calculate({ total: '5', next: '2', operation: '+' }, '0')).toEqual({
+      total: '5',
+      next: '20',
+      operation: '+',
+    });
+  });
+
+  it('updates next when given "."', () => {
+    expect(calculate({ total: '5', next: '2', operation: '+' }, '.')).toEqual({
+      total: '5',
+      next: '2.',
+      operation: '+',
+    });
+  });
+
+  it('updates the operation when given an operation button', () => {
+    expect(calculate({ total: '5', next: '2', operation: '+' }, '-')).toEqual({
```

Comment:
- [ ] **Good job so far for testing the files `operate.js` and `calculate.js` using Jest👏 👏** However, There are some failing tests that would be better to fix 👍 👍 
   ![image](https://user-images.githubusercontent.com/102757126/236601073-5a5cb6a6-fe14-47bc-adc6-09659bf7f437.png)
 
- [ ] In this project, we were also suggested to test all React components using Jest snapshots to test the components, Kindly add these tests to complete the project requirements 🙏 you may follow this [link ](https://jestjs.io/docs/snapshot-testing)for more info 😎